### PR TITLE
Update Node minor version

### PR DESCRIPTION
**Pull request by ace-n**
Some `@google-cloud` packages have a Node version constraint of `^8.13.0 || >=10.10.0`. This PR allows us to use those packages on this image.

### CODE REVIEW

File: builders/base/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN apt-get update -y && \
     rm /var/lib/apt/lists/*_*
 
 # Install the latest LTS release of nodejs
-RUN mkdir /nodejs && curl https://nodejs.org/dist/v8.12.0/node-v8.12.0-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+RUN mkdir /nodejs && curl https://nodejs.org/dist/v8.16.0/node-v8.16.0-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
```
Comment (grant) [inline builders/base/Dockerfile]: The comment states latest LTS.
Use this:
`https://nodejs.org/dist/v10.16.0/node-v10.16.0-linux-x64.tar.gz`

See:
https://nodejs.org/en/about/releases/

**Comment by grant** [comment]
Not sure if Jason is the best reviewer of this tool anymore. Added Justin to also look over.

**Comment by JustinBeckwith** [comment]
The client libraries team no longer has a dependency on this repo, so I don't really have a horse in this race :)

**Comment by grant** [comment]
Thanks for the update Justin.

**Pull request merged**